packages/oid4vci-demo-frontend/src/configs/sphereon.json

```diff
@@ -2,8 +2,8 @@
   "general": {
     "credentialName": "Sphereon Guest",
     "issueCredentialType": "GuestCredential",
-    "oid4vpAgentBaseUrl": "https://ssi.sphereon.com/demo/issuer",
-    "oid4vciAgentBaseUrl": "https://ssi.sphereon.com/demo/issuer",
+    "oid4vpAgentBaseUrl": "http://192.168.1.200:5000",
```

Comment:
This needs to be changed

Reply:
see my other comment

Reply:
this was done by the original version of the script. It's no longer overwriting source files